data/db_demo.yaml

```diff
@@ -30,9 +47,13 @@ groups/=program_A/users:
   - username: fulluser@cesium-ml.org
     admin: false
 
+groups/=program_A/streams:
+  - stream_id: =ztf_partnership
+
 filters:
-  - query_string: sample_query_string
+  - name: Yellow transients
```

Comment:
Should this be versioned?

Reply:
I wouldn't, but I tend to generally not care about such things :-) Are we doing that in any other situations?

Reply:
I thought we were keeping track of multiple filter versions. Is that only happening on the K side or should we do that in SP as well?

Reply:
Only on the K side, since that is where the action is happening -- I want to separate the concerns as much as possible and also reduce the amount of duplicated info to a bare minimum. SP really only stores `filter_id` and the corresponding `group_id`, and that is the only piece of information that is duplicated on K. That way, the AMPEL folks, for example, won't have to bother about filter versioning on the SP side either -- they would just create a filter and then post stuff to /candidates with the `filter_id`.